### ISSUE TITLE
Add zhimi.aifresh.ua1

### ISSUE THREAD

**Issue by fooltoy** [syssi/xiaomi_airpurifier#211]
Hello.
Do you plan to add control for Xiaomi Mijia Air Fresh C1-80 (mjxfj - 80 - g3) ?
Thanks for your hard work.

**Comment by syssi** [comment]
It looks like the device is pretty new. It's not listed here: https://miot-spec.org/miot-spec-v2/instances?status=all

**Comment by DenneyTang** [comment]
> It looks like the device is pretty new. It's not listed here: https://miot-spec.org/miot-spec-v2/instances?status=all

it's supported already

"status":"released",**"model":"zhimi.airfresh.ua1"**,"version":1,"type":"urn:miot-spec-v2:device:air-fresh:0000A012:zhimi-ua1:1"},{"status":"debug","model":"avc.magnet.winlk1","version":1,"type":"urn:miot-spec-v2:device:magnet-sensor:0000A016:avc-winlk1:1"},

**Comment by SEVSM** [comment]
The question is still relevant. Are there any plans to add control for UA1?
Thank you!

**Comment by theodorx7** [comment]
This is a popular ventilator model. It would be really cool if its support was implemented :)

**Comment by darkbaton22** [comment]
Yes, I would really like to have the support of the device in the Home Assistant

**Comment by SergGrib** [comment]
I join the request, I would like to see support for this model in the Home Assistant.
or would anyone suggest how to modify the code, if it is of course possible for this model, based on a similar

**Comment by Alex174-HA** [comment]
please add support for this model to the integration to НА

**Comment by mrmaximas** [comment]
@syssi Hello. I discovered that my two zhimi.aifresh.ua1 are not supported. Is there any chance of adding them? Thanks!
